pallets/liquidity-pools-gateway/queue/src/lib.rs

```diff
@@ -130,7 +131,7 @@ pub mod pallet {
 		/// are not reverted.
 		///   - an extra defensive weight is added in order to cover the weight
 		/// used when processing the message.
-		#[pallet::weight(T::WeightInfo::process_message().saturating_add(Weight::from_parts(200_000_000, 4096)))]
+		#[pallet::weight(T::WeightInfo::process_message().saturating_add(Weight::from_parts(LP_DEFENSIVE_WEIGHT_REF_TIME, LP_DEFENSIVE_WEIGHT_POV)))]
```

Comment:
NIT. Maybe the constant could be the own `Weight::from_parts(LP_DEFENSIVE_WEIGHT_REF_TIME, LP_DEFENSIVE_WEIGHT_POV)` (?)